fastlane/fastlane
# [frameit] Update iPad 12.9 version renaming mapping to match what the rest of frameit expects

### Checklist

### Motivation and Context

When I set up my Framefile I was all excited. One command to create all my screenshots?! I ran it, it downloaded the frames, and started its job. Until it was rudely interrupted by a missing frame.

<img width="1057" alt="Screen Shot 2021-03-23 at 5 11 55 PM" src="https://user-images.githubusercontent.com/997637/112227130-22194400-8bfd-11eb-9bdd-0d75b1e4644f.png">

### Description

Spelunking in the `~/.fastlane/frameit/latest` directory led me to realize that the iPad renaming scheme set the iPads to `(3rd generation)` and the rest of the frameit code was looking for `(4th generation)`. This updates the Rakefile used to rename iPad assets (among other things) to use "4th generation". Additionally, the files from Facebook don't include quotation marks and this updates the mapping to not expect them.

A companion PR in the frameit-frames repo is here: https://github.com/fastlane/frameit-frames/pull/21

### Testing Steps

From a fresh install (or after removing the `~/.fastlane/frameit/latest` directory)*, download frames either explicitly or just as a byproduct of running `frameit`. If you have an iPad Pro 12.9"-sized image (2732 x 2048) in your directory, `frameit` should frame it without the error shown above.

*To test before the companion PR is merged, you can point to `HOST_URL = "https://thillsman.github.io/frameit-frames"` in `frameit/lib/frameit/frame_downloader.rb` to get the frames created by the Rakefile modified here.
